editor/src/components/canvas/canvas-strategies/escape-hatch-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,308 @@
+      EP.appendNewElementPath(TestScenePath, ['aaa', 'ccc', 'ddd', 'fff', 'hhh']),
+      EP.appendNewElementPath(TestScenePath, ['aaa', 'ccc', 'ddd', 'fff']),
+    ]
+    await renderResult.dispatch([runEscapeHatch(targets)], true)
+
+    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
+      makeTestProjectCodeWithSnippet(`
+        <div style={{ width: '100%', height: '100%' }} data-uid='aaa'>
+          <div
+            style={{ width: 80, height: 80 }}
+            data-uid='bbb'
+          />
+          <div
+            style={{
+              display: 'flex',
+              padding: 8,
+              gap: 8,
+              flexDirection: 'column',
+            }}
+            data-uid='ccc'
+          >
+            <div
+              style={{
+                display: 'flex',
+                gap: 70,
+                flexDirection: 'row',
+                justifyContent: 'space-between',
+                alignItems: 'baseline',
+                position: 'absolute',
+                left: 8,
+                width: 384,
+                top: 88,
+                height: 63,
+              }}
+              data-uid='ddd'
+            >
+              <div data-uid='eee'>Hello Text</div>
+            </div>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                gap: 15,
+                position: 'absolute',
+                left: 343,
+                width: 49,
+                top: 88,
+                height: 59,
+              }}
+              data-uid='hhh'
+            >
+              <div
+                style={{
+                  backgroundColor: '#0091FFAA',
+                  height: 15,
+                  width: 15,
+                  border: '1px solid rgb(0, 0, 0, 1)',
+                }}
+                data-uid='iii'
+              />
+              <div
+                style={{
+                  backgroundColor: '#0091FFAA',
+                  border: '1px solid rgb(0, 0, 0, 1)',
+                  width: 15,
+                  height: 15,
+                }}
+                data-uid='jjj'
+              />
+            </div>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                gap: 15,
+                position: 'absolute',
+                left: 272,
+                width: 120,
+                top: 88,
+                height: 59
+              }}
+              data-uid='fff'
+            >
+              <div
+                style={{
+                  height: 57,
+                  width: 54,
+                  border: '1px solid rgb(0, 0, 0, 1)',
+                }}
+                data-uid='ggg'
+              />
+            </div>
+        </div>
+      </div>
+      `),
+    )
+  })
+  /* eslint-disable @typescript-eslint/no-empty-function */
+  xit('Converts element with absolute descendants', async () => {})
```

Comment:
Is this a TODO?

Reply:
yes it's a TODO to not forget writing tests after fixing the other bug with absolute elements